### DISCUSSION TITLE
DAI 2024-2025 - Check out my GitHub profile! - Gonçalo Heleno

### DISCUSSION THREAD

**Discussion by lentidas** [heig-vd-dai-course/heig-vd-dai-course#239] [Show and tell]
https://github.com/lentidas

**Comment by ludelafo** [comment]
Nice and complete GitHub README profile! Thank you!

For how long have you been working at Camptocamp? Seems quite an experience to work there!

If you see elements in the course that you feel are not aligned with your professional experience, feel free to share them and I'd be glad to update the course.

**Reply by lentidas** [reply]
Thank you for the compliment! 😊

I've been working at Camptocamp for 3 years. It's been a great experience working there, I've learned a lot working with K8s and things like that.

And thank you for your openness to suggestions. If I see anything, I'll try and make constructive suggestions 😉
